docs/content/stable/explore/query-1-performance/_index.md

```diff
@@ -9,7 +9,7 @@ menu:
   stable:
     identifier: query-tuning
     parent: explore
-    weight: 280
+    weight: 300
 type: indexpage
```

Comment:
Child pages in this section now have broken links to pages that moved. Do a global search for the pages and fix